pkg/linkdiscovery/controller.go

```diff
@@ -242,6 +286,7 @@ func (c *Controller) enterLinkDiscovery() {
 	tPorts := time.NewTicker(time.Duration(c.config.PortRediscoveryFrequency) * time.Second)
 	tPrune := time.NewTicker(time.Duration(c.config.LinkPruneFrequency) * time.Second)
 
+	// Do I have to emit ARP packets here? I guess so...
```

Comment:
I don't think so, unless we want to provoke the hosts to reply.

Reply:
Left out